CassandraStore.js

```diff
@@ -135,7 +135,7 @@ function CassandraStore(uri, callback) {
             .then(info => self.putInfoAsync(info.data))
             .then(() => self);
     }).catch(err => {
-        self._closeAsync().finally(() => { throw err; });
+        self.closeConnectionAsync().finally(() => { throw err; });
```

Comment:
i just noticed that this is a bug -- add `return` to this statement, or else it is not participating in the promise chain.